src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/CodeActions/CSharp/TypeAccessibilityCodeActionProviderTest.cs

```diff
@@ -207,8 +206,11 @@ public async Task Handle_ValidDiagnostic_InvalidCodeAction_ReturnsEmpty()
             Assert.Empty(results);
         }
 
-        [Fact]
-        public async Task Handle_ValidDiagnostic_ValidCodeAction_ReturnsCodeActions()
+        [Theory]
+        [InlineData("CS0246")]
+        [InlineData("CS0103")]
```

Comment:
It seems like support for `CS0103` existed before this PR and you're just filling in a missing test with this?

Reply:
Yep, and adding for `IDE1007`